ir_sensor_node:
  ros__parameters:
    ir_sensor_pin: 17


speaker_node:
  ros__parameters:
    servo_pin: 18
    audio_file: /home/liam/Downloads/M1F1-Alaw-AFsp.wav
    wave_count: 3
    wave_delay: 0.3
    servo_pulse_center: 1500
    servo_pulse_left: 1200
    servo_pulse_right: 1800
    